src/main/java/nl/tudelft/contextproject/gui/PresetController.java

```diff
@@ -1,34 +1,116 @@
 package main.java.nl.tudelft.contextproject.gui;
 
+import java.io.IOException;
```

Comment:
I updated my checkstyle and the errors now show up in my eclipse. I will fix them right away!